src/main/java/org/folio/orders/utils/ResourcePathResolver.java

```diff
@@ -10,48 +10,38 @@
   private ResourcePathResolver() {
   }
 
-  public static final String PO_LINES = "po_lines";
-  public static final String PO_NUMBER = "po_number";
-  public static final String PO_LINE_NUMBER = "po_line_number";
+  public static final String PO_LINES = "poLines";
+  public static final String PO_NUMBER = "poNumber";
+  public static final String PO_LINE_NUMBER = "poLineNumber";
 
   public static final String ADJUSTMENT = "adjustment";
   public static final String ALERTS = "alerts";
   public static final String CLAIMS = "claims";
   public static final String COST = "cost";
   public static final String DETAILS = "details";
   public static final String ERESOURCE = "eresource";
-  public static final String FUND_DISTRIBUTION = "fund_distribution";
+  public static final String FUND_DISTRIBUTION = "fundDistribution";
   public static final String LOCATIONS = "locations";
   public static final String PHYSICAL = "physical";
-  public static final String REPORTING_CODES = "reporting_codes";
+  public static final String REPORTING_CODES = "reportingCodes";
   public static final String SOURCE = "source";
-  public static final String VENDOR_DETAIL = "vendor_detail";
-  public static final String PURCHASE_ORDER = "purchase_order";
+  public static final String VENDOR_DETAIL = "vendorDetail";
+  public static final String PURCHASE_ORDER = "purchaseOrder";
   public static final String PIECES = "pieces";
   public static final String RECEIVING_HISTORY = "receiving-history";
-  public static final String RECEIPT_STATUS = "receipt_status";
+  public static final String RECEIPT_STATUS = "receiptStatus";
 
 
   private static final Map<String, String> SUB_OBJECT_ITEM_APIS;
   private static final Map<String, String> SUB_OBJECT_COLLECTION_APIS;
 
   static {
     Map<String, String> apis = new HashMap<>();
-    apis.put(ADJUSTMENT, "/orders-storage/adjustments");
     apis.put(ALERTS, "/orders-storage/alerts");
-    apis.put(CLAIMS, "/orders-storage/claims");
```

Comment:
You might consider to remove unused constants as well...

Reply:
Removed